website/source/docs/guides/managing-acl-policies.html.md

```diff
@@ -0,0 +1,150 @@
+
+### Operator-Only Implementation Example 
+
+In this following example Operators retain responsibility for service token management, but delegate access control between Connect-enabled services to the security team. 
+
+-> Note: The service registration examples describe the token generation process for per-service tokens, therefore, they are only applicable if you are able to create per-service tokens. 
+
+Initially, the Operator creates a single token with intention management privileges for the Security Team, and another service token for the Developer. For intention management the `intention` policy disposition should be included in the service rule.
+
+```hcl
+service "wordpress" 
+  { policy = "read" 
+    intentions = "write"
+  }
+```
+
+This enables the security team to create an intention that allows the upstream proxy for `mysql` to send data to the destination, the downstream proxy for `wordpress`.
+
+```sh
+consul intention create wordpress mysql
+``` 
+
+The Security Team responsible for managing intentions would need to create allow intentions when the `default_policy` is set to `deny`, since deny all will be inherited from the ACL configuration.
+
+In this implementation type, Developers are responsible for requesting a token for their service. For a Connect enabled service, the Operator would need to create a policy that provides write privileges for the service and proxy, and read privileges for the service_prefix and node_prefix.
+
+```hcl
+service "mysql" {
+  policy = "write"
+}
+
+service "mysql-sidecar-proxy" {
+  policy = "write"
+}
+
+service_prefix "" {
+    policy = "read"
+}
+
+node_prefix "" {
+    policy = "read"
+}
+```
+
+With the token the Developer can then register the service with Consul using the `register` command.
+
+```sh
+$ consul services register mysql
+``` 
+
+The Operator example above illustrates creating policies on the security spectrum. The first example, using an exact match resource rule, is the most secure. It grants the least privileges necessary to accomplish the task. Generally, creating policies and tokens with the least privileges will result in more policy definitions. However, this will help you create the most secure environment. Alternatively, the prefix rule can apply to zero-to-many objects. The trade-off of a less complicated token creation process is security. Note, this applies to all rules not just for agents.  
+
+## Next Steps
+
+After setting up access control processes, you will need to implement a token rotation policy. If you are using third-party tool to generate tokens, such as Vault, Consul ACL tokens will adhere to the TTLs set in that third party tool. If you are manually rotating tokens or need to revoke access, you can delete a token at any time with the [API](https://www.consul.io/api/acl/tokens.html#delete-a-token). 
```

Comment:
Worth noting here that if you do use automation to rotate tokens, then the application or tooling also needs to re-register with the local agent whenever the token changes.

Currently we don't document that anywhere so people who use the Vault integration will have their tokens expire after 30 days but typically never re-register with a new one meaning the consul agent actually looses the ability to register updates or update health status for the instance! In Connect world this is even worse as your certificates will not be able to be renewed after this and all traffic will break etc!

We should at least mention here when talking about rotation that currently rotation requires re-regristration with the new token to provide the local agent with a refreshed token to continue to work with.